packages/style-loader/README.md

```diff
@@ -190,6 +184,45 @@ module.exports = {
 Due to the inferred loader names, we highly recommend you stick to using objects
 to define loaders.
 
+### Loader Configuration
+
+You can configure loaders directly by passing `options`.
```

Comment:
Is it possible to pass options without a `postcss.config.js` file?

Reply:
Yeah you can do it right on the options object (instead of passing `config`): https://github.com/postcss/postcss-loader#options